.gitignore

```diff
@@ -1,6 +1,5 @@
-# Generated by Cargo
-# will have compiled files and executables
-/target/
-
-# These are backup files generated by rustfmt
+/target
+/.cargo
+/.ccls-cache
+/powerpc-unknown-eabi.json
```

Comment:
In future version of this repository, ``powerpc-unknown-eabi.json`` will be required for compiling examples built with ``ogc-rs``. Would it be possible for you to post your ``config.toml`` file in ``.cargo``? They might be useful for development, thanks!

Reply:
Fixed by removing `/powerpc-unknown-eabi.json` from `.gitignore`.